tools/cli/adminCommands.go

```diff
@@ -769,3 +806,49 @@ func AdminRefreshWorkflowTasks(c *cli.Context) {
 		fmt.Println("Refresh workflow task succeeded.")
 	}
 }
+
+func printAdminNamespace(resp *adminservice.DescribeNamespaceResponse) {
```

Comment:
the printing is going to be switched an automatic table printing tool similar to here once i start incorporating the UX improvements into admin commands https://github.com/temporalio/tctl/blob/382ae49c3dec490b8fcb5a353f36843f99597a01/cli/namespaceCommands.go#L370